docs/development/setup_development.rst

```diff
@@ -301,4 +324,4 @@ To configure a multi-machine evironment in Qubes, follow the Quick Start instruc
 create a standalone VM named ``sd-dev``, then follow the Linux instructions above to install the
 required packages, *omitting* Virtualbox.
 
-Then, complete the steps described in :doc:`qubes_staging`.
+Then, complete the steps described in :doc:`qubes_staging`.
```

Comment:
Couldn't figure out what changed here, but after checking out locally, it's just a no-trailing-newline change. :+1: